server/src/services/check.service.ts

```diff
@@ -13,50 +12,53 @@ export async function getCheckersWithMaxScore(taskId: number) {
     .select('t.name', 'taskName')
     .addSelect('"checkerUser"."githubId"', 'checkerGithubId')
     .addSelect('"studentUser"."githubId"', 'studentGithubId')
-    .addSelect('round("studentAvg"."avg", 1)', 'studentAvgScore')
+    .addSelect(
+      `
+      CASE
+        WHEN "studentScoreSumCnt"."cnt" > 2
+        THEN ROUND(("studentScoreSumCnt"."sum" - ts."score") / ("studentScoreSumCnt"."cnt" - 1), 1)
+        ELSE NULL
+      END
+    `,
+      'avg_excl_checker',
```

Comment:
could you please leave some comment above explaining logic here? just to make it more readable